test-tool/src/constants/regexp.js

```diff
@@ -58,6 +58,10 @@ module.exports = {
                 existence: 'missed local image',
             }
         },
+        localFileLink: {
+            regexp: /\[[^\]]+\]\((?!http)([a-z\-_A-Z0-9]+?)\.[a-z]{2,10}\)/,
+            message: 'incorrect link to local file, use full link to file on GitHub',
```

Comment:
@thecodester Could you please tell what the error message should be in case user specifies a link like (link title)[filename.ext] ?

Reply:
Incorrect link to local file, use full link to file on GitHub (starting with https://)

Reply:
Thank you

Reply:
Changed the message